build/tools/templates/Archive/Archive.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(DotNetRestoreSourcePropsPath)" Condition="'$(DotNetRestoreSourcePropsPath)' != ''"/>
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>$(ArchiveTargetFramework)</TargetFramework>
```

Comment:
Did you consider multi-targeting this project instead?
```
<TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
```

Reply:
Requires conditional metapackage references but yes that would work.